### PR TITLE
Add Ceph storage and migration plan

### PR DESCRIPTION
**Pull request by sellth**

### CODE REVIEW

File: bih-cluster/docs/storage/storage-migration.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+### Snapshots and Mirroring
+Snapshots are incremental copies of the state of the data at a particular point in time. 
+They provide safety against various "Oups, did I just delete that?" scenarios, meaning they can be used to recover lost or damaged files.
+
+Depending on the location and Tier, Cephfs utilises snapshots in differ differently.
+Some parts of Tier 1 and Tier 2 snapshots are also mirrored into a separate fire compartment within the datacenter to provide an additional layer of security.
+
+| Tier | Location                 | Path                         | Retention policy                | Mirrored |
+|:-----|:-------------------------|:-----------------------------|:--------------------------------|---------:|
+|    1 | User homes               | `/data/cephfs-1/home/users/` | Hourly for 48 h, daily for 14 d | yes      |
```
Comment (ericblanc20) [inline bih-cluster/docs/storage/storage-migration.md:34]: Do we need to mention it is backed-up (assuming it is, of course)?

File: bih-cluster/docs/storage/storage-migration.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- General purpose caching: `.cache` & `.local`
+- Containers: `.singularity` & `.apptainer`
+- Programming languanges libraries & registry: `R`, `.theano`, `.cargo`, `.cpan`, `.cpanm`, & `.npm`
+- Others: `.ncbi`, `.vs`
+
+!!! warning
+    Automatic symlink creation will not create a symlink to any conda installation.
+
+### Groups
+- Work on Tier 1: `/data/cephfs-1/work/groups/<group>`
```
Comment (ericblanc20) [inline bih-cluster/docs/storage/storage-migration.md:91]: No more `home` for groups & projects, I assume. Is that correct?

File: bih-cluster/docs/storage/storage-migration.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+Snapshots are incremental copies of the state of the data at a particular point in time. 
+They provide safety against various "Oups, did I just delete that?" scenarios, meaning they can be used to recover lost or damaged files.
+
+Depending on the location and Tier, Cephfs utilises snapshots in differ differently.
+Some parts of Tier 1 and Tier 2 snapshots are also mirrored into a separate fire compartment within the datacenter to provide an additional layer of security.
+
+| Tier | Location                 | Path                         | Retention policy                | Mirrored |
+|:-----|:-------------------------|:-----------------------------|:--------------------------------|---------:|
+|    1 | User homes               | `/data/cephfs-1/home/users/` | Hourly for 48 h, daily for 14 d | yes      |
+|    1 | Group/project work       | `/data/cephfs-1/work/`       | Four times a day, daily for 5 d | yes      |
```
Comment (ericblanc20) [inline bih-cluster/docs/storage/storage-migration.md:35]: I had this un-mirrored. I assume it was a mistake,wasn't it?

File: bih-cluster/docs/storage/storage-migration.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+|    1 | Group/project scratch    | `/data/cephfs-1/scratch/`    | Daily for 3 d                   | no       |
```
Comment (ericblanc20) [inline bih-cluster/docs/storage/storage-migration.md:36]: Do we need to mention that the data will be erased after 14 days (assuming it is still the case)?

**Review by ericblanc20 — COMMENTED** [review]
Looks very good. There are a couple of things I am unsure of, especially regarding the backup/snapshot policy.

File: bih-cluster/docs/storage/storage-migration.md (hunk 1 above)
Comment (sellth) [reply bih-cluster/docs/storage/storage-migration.md:34]: Are there actually backups besides the mirrored snapshots? I'm not sure, we have to clarify.

File: bih-cluster/docs/storage/storage-migration.md (hunk 4 above)
Comment (sellth) [reply bih-cluster/docs/storage/storage-migration.md:36]: This is not the place to explain scratch imo. We should have an entire section on that (maybe it already exists?).

**Review by sellth — COMMENTED** [review]
lgtm

File: bih-cluster/docs/storage/migration-faq.md [hunk 5]
```diff
@@ -24,6 +24,9 @@ $ rm -r $SOURCE
     When defining your source location, do not use the `*` wildcard character.
     This will not match hidden (dot) files and leave them behind.
 
+!!! Note
+	Paranoid users may want to consider using `hashdeep` to ensure that all files were successfully copied.
+
```
Comment (sellth) [inline bih-cluster/docs/storage/migration-faq.md:29]: we should probably recommend `rsync --checksum` flag

Comment (sellth) [reply bih-cluster/docs/storage/migration-faq.md:29]: Re-write second code block to include. Hashdeep as 2nd option.

**Pull request merged**